cnf/ext/repositories.bnd

```diff
@@ -34,15 +34,16 @@ mavencentral:           https://repo.maven.apache.org/maven2
 
 -plugin.1.Eclipse:\
     aQute.bnd.repository.osgi.OSGiRepository;\
-        name="Eclipse Oxygen 4.7.3a";\
-        locations="https://dl.bintray.com/bndtools/eclipse-repo/4.7.3a/index.xml.gz";\
+        name="Eclipse 2018-12";\
+        locations="https://dl.bintray.com/bndtools/eclipse-repo/4.10/index.xml.gz";\
         poll.time=-1;\
-        cache="${workspace}/cnf/cache/stable/EclipseOxygen",\
+        max.stale=-1;\
```

Comment:
What is the point of this? The default is already one year wish should be sufficient for most uses.

Reply:
For some reason it was downloading bundles from this repo too often for my taste and I hoped this would stop it. I've removed it now.

Reply:
@juergen-albert , if you're running an older version of Bnd, you may have been hitting #3256 (which @pkriens has since fixed).